packages/documentation/src/content/docs/admin/liquidity/asset-liquidity.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: Asset liquidity
+tableOfContents:
+  maxHeadingLevel: 4
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components'
+import { CodeBlock, LinkOut } from '@interledger/docs-design-system'
+import IdempotencyNote from '/src/partials/liquidity-idempotency.mdx'
+
+Asset liquidity is the amount of value, denominated in a given asset, that Rafiki has available to handle cross-currency (foreign exchange) transactions between you and your peer. Whenever an outgoing payment/incoming payment is in a different asset than the peering relationship, the liquidity of asset accounts will change depending on the FX direction.
```

Comment:
ah, we should also update the `overview/concepts/accounting/#asset-liquidity-accounts` page with this text for consistency

Reply:
I caught that yesterday, so I think we're good here.